src/transit_equity/census/household_size.py

```diff
@@ -0,0 +1,131 @@
+    B11016_010E: HouseholdSizeDetails = HouseholdSizeDetails(field='B11016_010E',
+        label='nonfamily_1_person_household', count=1)
+    B11016_011E: HouseholdSizeDetails = HouseholdSizeDetails(field='B11016_011E',
+        label='nonfamily_2_person_household', count=2)
+    B11016_012E: HouseholdSizeDetails = HouseholdSizeDetails(field='B11016_012E',
+        label='nonfamily_3_person_household', count=3)
+    B11016_013E: HouseholdSizeDetails = HouseholdSizeDetails(field='B11016_013E',
+        label='nonfamily_4_person_household', count=4)
+    B11016_014E: HouseholdSizeDetails = HouseholdSizeDetails(field='B11016_014E',
+        label='nonfamily_5_person_household', count=5)
+    B11016_015E: HouseholdSizeDetails = HouseholdSizeDetails(field='B11016_015E',
+        label='nonfamily_6_person_household', count=6)
+    B11016_016E: HouseholdSizeDetails = HouseholdSizeDetails(field='B11016_016E',
+        label='nonfamily_7_or_more_person_household', count=7)
+
+# The main motive of this dictionary is to map the column names to human readable names
+HOUSEHOLD_SIZE_COLUMNS_DICT = {
+    'B11016_001E': 'total',
+    'B11016_002E': 'family_households',
+    'B11016_003E': 'family_2_person_household',
+    'B11016_004E': 'family_3_person_household',
+    'B11016_005E': 'family_4_person_household',
+    'B11016_006E': 'family_5_person_household',
+    'B11016_007E': 'family_6_person_household',
+    'B11016_008E': 'family_7_or_more_person_household',
+    'B11016_009E': 'nonfamily_households',
+    'B11016_010E': 'nonfamily_1_person_household',
+    'B11016_011E': 'nonfamily_2_person_household',
+    'B11016_012E': 'nonfamily_3_person_household',
+    'B11016_013E': 'nonfamily_4_person_household',
+    'B11016_014E': 'nonfamily_5_person_household',
+    'B11016_015E': 'nonfamily_6_person_household',
+    'B11016_016E': 'nonfamily_7_or_more_person_household',
+}
+
+# TODO: This function can be extended to do a calculation on an entire pandas DataFrame
+# The extended function will also have better performance.
+def get_average_household_size_from_census_row(census_row: pd.Series) -> float:
+    '''
+    A function to get the average household size from a census row
+
+    Parameters:
+    ----------
+    census_row: pd.Series
+        A pandas series containing the census data
+        Should contain all the columns in transit_equity.census.household_size.HOUSEHOLD_SIZE_COLUMNS
+    
+    Returns:
+    -------
+    float
+
+    Examples:
+    --------
+    Example 1:
+    >>> import pandas as pd
+    >>> from transit_equity.census.household_size import HOUSEHOLD_SIZE_COLUMNS
+    >>> from transit_equity.census.household_size import get_average_household_size_from_census_row
+    >>> census_row_dict = dict()
+    >>> for column in HOUSEHOLD_SIZE_COLUMNS:
+    ...     census_row_dict[column.value.field] = 0
+    >>> census_row = pd.Series(census_row_dict)
+    >>> get_average_household_size_from_census_row(census_row)
```

Comment:
I guess this specific example may not be the most useful since it only mentions an edge case.
Is this still fine? Any thoughts @bhazelton ?

Reply:
Maybe write something explaining what the example is doing? I can't quite tell the purpose.

Reply:
I believe the new comments should add enough clarity. 
Please do let me know @bhazelton

Reply:
This is great, thanks!